### PR TITLE
Last minor fixes v1.7.1

### PR DESCRIPTION
**Pull request by SovereignAndrey**
Ticket(s):
- #1802 

## Explanation of the solution
- new main logo SVGs added
- new footer links added/removed/updated
- fix minor lint warning
- New header setup

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp

#### New Header
image

#### New Footer
image

### CODE REVIEW

**Review by ap-justin — APPROVED** [review]

File: src/App/__tests__/App.test.tsx [hunk 1]
```diff
@@ -54,21 +54,10 @@ describe("App.tsx tests", () => {
     const footer = screen.getByRole("contentinfo");
     expect(footer).toBeInTheDocument();
 
-    expect(
-      screen.getByRole("link", {
-        name: marketLink,
-      })
-    ).toBeInTheDocument();
-    expect(
```
Comment (misicnenad) [inline src/App/__tests__/App.test.tsx:62]: Any tests related to navigating to `Leaderboard` or `Register` should be removed, as those links are no longer part of the header (thus causing the tests to fail).

**Review by misicnenad — CHANGES_REQUESTED** [review]

File: src/App/__tests__/App.test.tsx [hunk 2]
```diff
@@ -32,8 +32,6 @@ jest.mock("services/aws/leaderboard", () => ({
 const heroText = /Reflecting on the past/i;
 const leadText = /leaderboard/i;
```
Comment (misicnenad) [inline src/App/__tests__/App.test.tsx:33]:
```suggestion
```

**Review by misicnenad — CHANGES_REQUESTED** [review]
Lint errors present. We can approve once these are addressed.

**Comment by SovereignAndrey** [comment]
> Lint errors present. We can approve once these are addressed.

It says all checks passing good ser.

**Pull request merged**